notebooks/samples/explanations/tf2/ai-explanations-image.ipynb

```diff
@@ -95,7 +95,7 @@
       "source": [
```

Comment:
<p>Here's another example - use the same present tense verb form when you're describing what each method does. I cut down each sentence to the verb to illustrate this:</p><p> </p><ul><li><code>cache()</code>&nbsp;- Loads...</li><li><code>shuffle(2048)</code>&nbsp;- Randomly shuffles...</li><li><code>repeat()</code>&nbsp;- Allows...</li><li><code>batch(BATCH_SIZE)</code>&nbsp;- Sets...</li><li><code>prefetch(AUTO)</code>&nbsp;- Prefetches...</li></ul><p> </p><p> </p><p>Typo: "duringg"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>